BookReader/BookReader.js

```diff
@@ -627,7 +627,7 @@ BookReader.prototype.drawLeafsOnePage = function() {
         var topInView    = (leafTop >= scrollTop) && (leafTop <= scrollBottom);
         var bottomInView = (leafBottom >= scrollTop) && (leafBottom <= scrollBottom);
         var middleInView = (leafTop <=scrollTop) && (leafBottom>=scrollBottom);
-        if (topInView | bottomInView | middleInView) {
```

Comment:
In this case the situation is simple: all three variables contain boolean values (see lines 627 ff),
so using `||` is fine.